client/state/posts/test/actions.js

```diff
@@ -153,4 +163,70 @@ describe( 'actions', () => {
 			} ).catch( done );
 		} );
 	} );
+
+	describe( '#requestSitePost()', () => {
+		before( () => {
+			nock( 'https://public-api.wordpress.com:443' )
```

Comment:
not for this PR, but if it's too painful to mock wpcom.js, maybe we should look at fixing wpcom.js to make it easier to test through. Having to bake all this info (host / port / protocol / path) into tests feels fragile.